cl/scrapers/utils.py

```diff
@@ -202,6 +202,11 @@ def get_binary_content(
             msg = f"EmptyFileError: {download_url}\n{traceback.format_exc()}"
             return msg, None
 
+        # test for captcha redirects
```

Comment:
From the `nev` issue that inspired this PR, I don't think this is triggered by a captcha. Also, the JS extension is gotten from doctor, not directly from the response, so the actual check for `.js` extension should be here
https://github.com/freelawproject/courtlistener/blob/main/cl/scrapers/management/commands/cl_scrape_opinions.py#L139

Check my comment in the [issue](https://github.com/freelawproject/courtlistener/issues/3695#issuecomment-1918107491) for more detail

Reply:
Fascinating- any chance you validated that this file was being identified in Doctor as a JS file?